osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/DeviceChannelsHelper.java

```diff
@@ -54,8 +54,12 @@
     private static final short FIRST_CHANNEL = 1;
     private static final short NR_OF_CHANNELS = 4;
 
+    private final DlmsHelper dlmsHelper;
+
     @Autowired
-    private DlmsHelper dlmsHelper;
+    public DeviceChannelsHelper(final DlmsHelper dlmsHelper) {
```

Comment:
what is the reason it was changed from injection to constructor param ?

Reply:
This way it was easier for the test I created. Otherwise the calls to the DlmsHelper needs to be mocked, causing a lot of work compared to the small change I made.